src/modules/locations.ts

```diff
@@ -0,0 +1,4 @@
+export const locations = {
+  root: () => '/',
+  signIn: () => '/sign-in'
```

Comment:
Is the sign-in route used by all dapps?

Reply:
The builder, marketplace, and account use this location.